File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/DTOs/BalanceOfOutputDTO.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.ABI.FunctionEncoding.Attributes;
namespace StandardToken.MyContractName.DTOs
{
    [FunctionOutput]
    public class BalanceOfOutputDTO
    {
        [Parameter("uint256", "balance", 1)]
        public BigInteger Balance {get; set;}
    }
}

File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/BalanceOfOutputDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[FunctionOutput]>
    Public Class BalanceOfOutputDTO
    
        <[Parameter]("uint256", "balance", 1)>
        Public Property Balance As BigInteger
    
    End Class

End Namespace
